internal/cli/command_factory.go

```diff
@@ -106,6 +107,14 @@ func (factory *CommandFactory) Build(command CommandDefinition) *cobra.Command {
 			factory.telemetryService.TrackEvent(telemetry.EventTypeCommandStart)
 
 			err := command.Command.Handler(factory.profile, factory.ui)
+			if err == realm.ErrInvalidSession {
+				factory.profile.ClearSession()
+				profileErr := factory.profile.Save()
+				if profileErr != nil {
+					//nolint:errcheck
```

Comment:
instead of the `nolint` here, you can print the original `profileErr` with `factory.errLogger.Printf` and then proceed along